tests/lib/devicetree/src/main.c

```diff
@@ -683,8 +683,7 @@ static void test_phandles(void)
 			     "TEST_GPIO_2"),
 		     "gpios[1].label");
 
-#if 0
```

Comment:
I didn't notice this! Sneaky sneaky :wink:

Reply:
I didn't either :)